spec/models/in_progress_form_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'support/attr_encrypted_matcher'
+
+RSpec.describe InProgressForm, type: :model do
+  describe 'form encryption' do
+    it { encrypts(:form_data) }
```

Comment:
Nitpicky, feel free to ignore.

This syntax doesn't quite match our other rspecs.  Could it read something like, 

```ruby
expect(subject).to encrypt_attr(:form_data)
```
It would require the `attr_encrypted_matcher.rb` to change also. I'm honestly not sure if my recommendation worse or better - just that the first time I read this line I was a little confused.